spec/lib/pre_assembly/file_manifest_spec.rb

```diff
@@ -180,7 +207,7 @@
                       filename: 'aa111aa1111_001_b_sh.wav', version: 1,
                       hasMessageDigests: [{ type: 'md5', digest: '0e80068efa7b0d749ed5da097f6d0eeb' }],
                       access: { view: 'world', download: 'none', controlledDigitalLending: false },
-                      administrative: { publish: false, sdrPreserve: false, shelve: false } },
+                      administrative: { publish: false, sdrPreserve: true, shelve: false } },
                     { type: 'https://cocina.sul.stanford.edu/models/file',
```

Comment:
changes go with existing fixtures that were updated above (this is invalid data now and cannot be created)